src/main/java/uk/gov/mca/beacons/service/dto/DomainDTO.java

```diff
@@ -8,6 +8,7 @@
 
   private UUID id;
   private final Map<String, Object> attributes = new HashMap<String, Object>();
+  private final Map<String, String> links = new HashMap<String, String>();
```

Comment:
in the json links look much the same as attributes, so lets make that a pattern :)